tools/gluten-te/ubuntu/tpc.sh

```diff
@@ -75,7 +75,7 @@ then
   exit 1
 fi
 
-DOCKER_SELECTED_TARGET_IMAGE_TPC_WITH_OS_IMAGE="$DOCKER_SELECTED_TARGET_IMAGE_TPC-$OS_IMAGE"
+DOCKER_SELECTED_TARGET_IMAGE_TPC_WITH_OS_IMAGE="$DOCKER_SELECTED_TARGET_IMAGE_TPC-$OS_IMAGE_NAME"
```

Comment:
We may do the same on tpc scripts to replaced `$OS_IMAGE` by `$OS_IMAGE:$OS_IMAGE_TAG`. But it's non-blocking and can be done in a separate ticket.

Reply:
Corrected. Still use $OS_IMAGE.